.github/workflows/packer_workflow_dispatch.yml

```diff
@@ -1,7 +1,7 @@
 name: Tessell Packer Wokflow Dispatch
 
-# Source branch is "Use workflow from" in "Run Workflow". We do not want this workflow to run when source branch is main or "rel-*".
-# BRANCH_NAME is " Provide branch name or commit id ". We only want to push to nexus when Branch_name is main or "rel-*".
+# PR_SOURCE_BRANCH is "Use workflow from" in "Run Workflow". This is the branch which will have new DB engines (so branch which will merge in main). We do not want this workflow to run when PR_SOURCE_BRANCH is main or "rel-*".
```

Comment:
@HritwikTessell  keep `SOURCE_BRANCH`. That is the variable name used across all CI code.
OK with `BASE_BRANCH` change.